core/src/database/clickhouse/generate.rs

```diff
@@ -186,24 +194,30 @@ pub fn generate_columns_with_data_types(inputs: &[ABIInput]) -> Vec<String> {
     generate_columns(inputs, &GenerateAbiPropertiesType::ClickhouseWithDataTypes)
 }
 
-pub fn drop_tables_for_indexer_clickhouse(project_path: &Path, indexer: &Indexer) -> Code {
+pub fn drop_tables_for_indexer_clickhouse(project_path: &Path, indexer: &Indexer, database_name: &str) -> Code {
     let mut sql = String::new();
 
-    sql.push_str("DROP TABLE IF EXISTS rindexer_internal.latest_block;");
+    sql.push_str(&format!("DROP TABLE IF EXISTS {}.rindexer_internal_latest_block;", database_name));
```

Comment:
To clarify this is because in hosted environemnt you want to just have `default` as the database and have tables be on the main db right